packages/component/src/Localization/yue.json

```diff
@@ -24,6 +24,12 @@
   "ACTIVITY_YOU_SAID_ALT": "你話：",
   "ADAPTIVE_CARD_ERROR_BOX_TITLE_PARSE": "Adaptive Card 解析出事",
   "ADAPTIVE_CARD_ERROR_BOX_TITLE_RENDER": "Adaptive Card render 出事",
+  "ATTACHMENT_AUDIO": "一段錄音",
```

Comment:
Aren't these supposed to be done by the localization team?

Reply:
"Yue" is community-contributed language.